test/python/tests/ana_client/docker-compose.yml

```diff
@@ -60,15 +62,19 @@ services:
     environment:
         - MY_POD_IP=10.0.0.4
         - NEXUS_NVMF_ANA_ENABLE=1
+        - NEXUS_NVMF_RESV_ENABLE=1
     command: ${SRCDIR}/target/debug/mayastor -g 0.0.0.0 -l 5,6 -r /tmp/ms2.sock
     networks:
       mayastor_net:
         ipv4_address: 10.0.0.4
     cap_add:
+      # NUMA related
       - SYS_ADMIN
       - SYS_NICE
+      # uring needs mmap
       - IPC_LOCK
     security_opt:
+      # we can set this to a JSON file to allow per syscall access
       - seccomp:unconfined
```

Comment:
As we're changing this file anyway, could you change this to `=` as this is in the system logs:
```
dockerd[1018]: time="2021-09-15T12:09:43.820452714+01:00" level=warning msg="Security options with `:` as a separator are deprecated and will be completely unsupported in 17.04, use `=` instead."
```

Reply:
Done